client/jetpack-connect/plans.jsx

```diff
@@ -267,6 +268,7 @@ class Plans extends Component {
 
 		return (
 			<div>
+				<DocumentHead title={ translate( 'Plans' ) } />
```

Comment:
There's an early-return `render` here as well. Should that render `DocumentHead`?